internal/controller/controllers/preprovisioningimage_controller.go

```diff
@@ -280,9 +285,17 @@ func (r *PreprovisioningImageReconciler) AddIronicAgentToInfraEnv(ctx context.Co
 		log.WithError(err).Error("failed to get corresponding infraEnv")
 		return ctrl.Result{}, err
 	}
+	ironicAgentImage := ""
+	if infraEnvInternal.OpenshiftVersion != "" {
```

Comment:
if i remember right for day2 installations we don't require OpenshiftVersion (capi too) should we warn user somehow?

Reply:
I guess I can add a default ironic agent platform.
I don't know if the user should care about the ironic agent image we will use.